src/components/pages/account/Header.tsx

```diff
@@ -24,7 +24,7 @@ export default function Header({
   account,
 }: HeaderProps): JSX.Element {
   const { data: parent } = useAccount(account.parentId);
-  const { data: splits } = useSplits({ guid: account.guid });
+  const { data: splits } = useSplitsPagination(account.guid);
```

Comment:
we only need the splits to know the latest date and if there are actual splits so using the same that is already loaded for the transactions table which reduces the amount of queries.